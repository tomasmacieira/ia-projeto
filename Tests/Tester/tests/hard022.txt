ROW	2	1	1	2	2	4	2	1	2	3
COLUMN	2	0	4	2	0	1	4	2	2	3
1
HINT	0	7	C
