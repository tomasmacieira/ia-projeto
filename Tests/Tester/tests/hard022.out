..t....C..
..m.......
..b.......
......t.c.
t.....m...
b..c..m..t
......b..m
.........b
..lr......
.....c.lr.
